ynr/apps/moderation_queue/views.py

```diff
@@ -544,6 +544,14 @@ class SuggestLockView(LoginRequiredMixin, CreateView):
     def form_valid(self, form):
         user = self.request.user
         form.instance.user = user
+
+        LoggedAction.objects.create(
+            user=self.request.user,
+            action_type="suggest-ballot-lock",
+            post_election=form.cleaned_data["postextraelection"],
+            source=form.cleaned_data["justification"],
```

Comment:
Can we also pass `ip_address=get_client_ip(self.request)` here?

Reply:
👍